### PR TITLE
Make RestrictedDataCache optional

### PR DESCRIPTION
**Pull request by ostcar**
I have not test it. Please test, fix, and merge it. Alternative you can review it and I fix it next week.

### CODE REVIEW

File: openslides/utils/settings.py.tpl [hunk 1]
```diff
@@ -112,6 +112,11 @@ if use_redis:
     SESSION_ENGINE = 'redis_sessions.session'
 
 
+# When use_redis is True, the restricted data cache caches the data individuel
+# for each user. This requires a lot of memory if there are a lot of active
+# users. If use_redis is False, this setting has no effect.
+USE_RESTRICTED_DATA_CACHE = True
```
Comment (normanjaeckel) [inline openslides/utils/settings.py.tpl]: Let us please use a falsy default: `DISABLE_USER_CACHE=False` Also we have to explain in three words what the restricted data cache is. Non developers won't know.

Comment (emanuelschuetze) [reply openslides/utils/settings.py.tpl]: Done.

**Comment by emanuelschuetze** [comment]
@ostcar I can't push into your branch. Can you check it? Otherwise I create a new pull request.

**Comment by emanuelschuetze** [comment]
> @ostcar I can't push into your branch. Can you check it? Otherwise I create a new pull request.

Sorry, my fault. Works now.

**Comment by emanuelschuetze** [comment]
@normanjaeckel Please review again.

**Comment by emanuelschuetze** [comment]
@FinnStutzenstein Please test it in your performance test setup with huge database. Thanks.

@ostcar Thanks a lot for your quick fix!

File: openslides/utils/cache.py [hunk 2]
```diff
@@ -296,7 +297,10 @@ def get_redis_connection() -> Any:
 
 if use_redis_cache():
     websocket_user_cache = RedisWebsocketUserCache()  # type: BaseWebsocketUserCache
-    restricted_data_cache = RestrictedDataCache()  # type: Union[RestrictedDataCache, DummyRestrictedDataCache]
+    if settings.get('DISABLE_USE_CACHE', False):
```
Comment (normanjaeckel) [inline openslides/utils/cache.py]: Typo `USER`

Comment (normanjaeckel) [reply openslides/utils/cache.py]: Is it also possible to to it as shown [here](https://docs.djangoproject.com/en/1.11/topics/settings/#using-settings-in-python-code)?

Comment (emanuelschuetze) [reply openslides/utils/cache.py]: Done.

**Comment by FinnStutzenstein** [comment]
Pushed in a travis fix.

**Comment by emanuelschuetze** [comment]
@normanjaeckel All done. Ready for your last review. As discuss, I have to merge it now. Feel free to remark something later. I marked this pull request with new label "needs review" for you.

**Comment by normanjaeckel** [comment]
Ok.

**Pull request merged**